mne/viz/evoked.py

```diff
@@ -477,7 +477,7 @@ def whitened_gfp(x, rank=None):
 
     if not has_sss:
         evokeds_white[0].plot(unit=False, axes=axes_evoked,
-                              hline=[-1.96, 1.96])
+                              hline=[-1.96, 1.96], show=show)
```

Comment:
with plot_white, current behavior, with no change, the plot appears when show=False. 
with change, the plots don't appear when show=False.